src/utils/extentUtils.js

```diff
@@ -33,3 +38,45 @@ export function getExtentForProjection(projection, extent) {
     }
     return null
 }
+
+/**
+ * @param {Number} config.size Number of pixels the extent should be (if 100 is given, a box of
+ *   100x100 pixels with the coordinate at its center will be returned)
+ * @param {[Number, Number]} config.coordinate Where the center of the 100pixel extent should be.
+ * @param {CoordinateSystem} config.projection Projection used to describe the coordinates
+ * @param {Number} config.resolution Current map resolution, necessary to calculate how much
+ *   distance 100pixel means.
+ * @param {Boolean} [config.rounded=false] Tells if the extent's value should be rounded before
+ *   being returned. Default is `false`
+ * @returns {[[Number, Number], [Number, Number]]} A squared extent of 100pixel around the given
+ *   coordinates
+ */
+export function createPixelExtentAround(config) {
+    const { size, coordinate, projection, resolution, rounded = false } = config
+    if (!size || !coordinate || !projection || !resolution) {
+        return null
+    }
+    let coordinatesWgs84 = coordinate
+    if (projection.epsg !== WGS84.epsg) {
+        coordinatesWgs84 = proj4(projection.epsg, WGS84.epsg, coordinate)
+    }
+    let extent = bbox(
+        buffer(
+            point(coordinatesWgs84),
+            // sphere of the wanted number of pixels as radius around the coordinate
+            size * resolution,
+            { units: 'meters' }
```

Comment:
I don't know the turf library but it seems strange to use `meter` with WGS84 ?

Reply:
this lib can do quite a lot of unit conversion